src/main/java/org/opensearch/ad/model/AnomalyDetector.java

```diff
@@ -330,6 +348,21 @@ public void writeTo(StreamOutput output) throws IOException {
         } else {
             output.writeBoolean(false);
         }
+        if (customResultIndexMinSize != null) {
+            output.writeOptionalInt(customResultIndexMinSize);
```

Comment:
You can remove null check as writeOptinalInt covers that.

Reply:
using `writeOptinalInt` in both AD and Config file